tools/bashreadline.bt

```diff
@@ -12,6 +12,22 @@
  * Copyright 2018 Netflix, Inc.
  *
  * 06-Sep-2018	Brendan Gregg	Created this.
```

Comment:
Nit: can we keep the copyright and changelog at the bottom (for all scripts) ?

Reply:
Sounds good!